### PR TITLE
Applied fixes from StyleCI

### PR DESCRIPTION
**Pull request by OskarStark**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/zDD4rn).

### CODE REVIEW

File: Admin/AbstractAdmin.php [hunk 1]
```diff
@@ -1648,7 +1648,7 @@ public function setSubject($subject)
 This is deprecated since 3.x and will no longer be supported in 4.0.
 EOT;
 
-            trigger_error(
+            @trigger_error(
```
Comment (greg0ire) [inline Admin/AbstractAdmin.php]: What happens without the `@` :s ? Are errors just displayed, like that ?

**Comment by OskarStark** [comment]
@greg0ire please lets merge #4108 instead of this PR

**Comment by OskarStark** [comment]
argh 😄

**Comment by greg0ire** [comment]
Woops sorry

**Comment by OskarStark** [comment]
np :)

**Pull request merged**